document-service-api/src/lib/config.js

```diff
@@ -19,6 +21,23 @@ module.exports = {
       },
     },
   },
+  docs: {
+    api: {
+      path: process.env.DOCS_API_PATH || path.join(__dirname, '..', 'api'),
+    },
+  },
+  fileUpload: {
+    maxSizeInBytes: Number(process.env.FILE_MAX_SIZE_IN_BYTES || 1000),
+    mimeTypes: [
+      'application/pdf', // pdf
+      'application/msword', // doc
+      'application/vnd.openxmlformats-officedocument.wordprocessingml.document', // docx
+      'image/tiff', // tiff
+      'image/jpeg', // jpeg
```

Comment:
could be image/jpg as well

Reply:
Seemingly not https://stackoverflow.com/questions/33692835/is-the-mime-type-image-jpg-the-same-as-image-jpeg